src/Helper/Settings.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace Drupal\os2forms_fasit\Helper;
+
+use Drupal\Core\KeyValueStore\KeyValueFactoryInterface;
+use Drupal\Core\KeyValueStore\KeyValueStoreInterface;
+use Drupal\os2forms_fasit\Exception\InvalidSettingException;
+use Drupal\os2forms_fasit\Form\SettingsForm;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * General settings for os2forms_fasit.
+ */
+final class Settings {
+  /**
+   * The store.
+   *
+   * @var \Drupal\Core\KeyValueStore\KeyValueStoreInterface
+   */
+  private KeyValueStoreInterface $store;
```

Comment:
I know where this is coming from, but we should really consider (again) using config to store module config (settings) …

Reply:
For now let's keep it this way (key-value).